pkg/vpcmodel/grouping.go

```diff
@@ -68,43 +68,47 @@ func newGroupingConnections() *groupingConnections {
 func newGroupConnLines(c *VPCConfig, v *VPCConnectivity,
 	grouping bool) (res *GroupConnLines, err error) {
 	res = &GroupConnLines{config: c, nodesConn: v,
-		srcToDst:                 newGroupingConnections(),
-		dstToSrc:                 newGroupingConnections(),
-		groupedEndpointsElemsMap: map[string]*groupedEndpointsElems{},
-		groupedExternalNodesMap:  map[string]*groupedExternalNodes{}}
+		srcToDst: newGroupingConnections(),
+		dstToSrc: newGroupingConnections(),
+		cacheGrouped: cacheGroupedElements{
+			map[string]*groupedEndpointsElems{},
+			map[string]*groupedExternalNodes{}}}
```

Comment:
change the type of `cacheGrouped cacheGroupedElements` within `GroupConnLines` to be `*cacheGroupedElements`, and then use `initCacheGroupedElem()` here as well.

Reply:
done